### DISCUSSION TITLE
Best IDE?

### DISCUSSION THREAD

**Discussion by PwrSrg** [tailwindlabs/tailwindcss#3673] [Help]
Hi all,

I am very **NEW** to Tailwind, and am looking for the **best IDE**.  The only two I have found so far are **PRICEY**!

[Shuffle](https://shuffle.dev) - ~~$149~~ $249 Lifetime - https://shuffle.dev
[Updrafts](https://updrafts.app) - $249 Lifetime - https://updrafts.app

I also found [this](https://github.com/crow1796/tailwind-studio), but the project looks **abandoned**. 

Like most people, I am coming from **Bootstrap**, and currently use **[Bootstrap Studio](https://bootstrapstudio.io)** for laying out my designs.

Thanks!

**EDIT 10/6/22** - Shuffle lifetime is no longer $149

**Comment by SyDant** [comment]
Hi, i see your idea, how about now situation ?? are you still using shuffle for tailwindcss or pinegrow ? or any new ? let me know
